src/Controller/BookController.php

```diff
@@ -76,9 +76,23 @@ public function add()
                 return "";
             } else {
                 $errors = $book->getErrors();
+                $bookArray = $bookManager->selectAll();
+                $bookArray['title'] = $_POST['title'];
+                $bookArray['author'] = $_POST['author'];
+                $bookArray['genre'] = $_POST['genre'];
+                $bookArray['localization'] = $_POST['localization'];
+                $bookArray['releaseDate'] = $_POST['releaseDate'];
+                $bookArray['description'] = $_POST['description'];
+                $bookArray['isbn'] = $_POST['isbn'];
+                $bookArray['hasBeenReadOn'] = $_POST['hasBeenReadOn'];
+                $bookArray['hasBeenRead'] = $read;
```

Comment:
If you want all the books and the uncompleted book from the form, you probably should have different variables.

And you should only keep the valid fields and not take everything from $_POST blindly